packages/api/lambdas/sf-sns-broadcast.js

```diff
@@ -64,38 +63,9 @@ function makeLambdaFunctionFail(event) {
 async function publish(message, finished = false) {
   const event = await pullStepFunctionEvent(message);
 
-  const topicArn = get(event, 'meta.topic_arn', null);
   const failed = eventFailed(event);
 
-  if (topicArn) {
-    // if this is the sns call at the end of the execution
-    if (finished) {
-      if (failed) {
-        event.meta.status = 'failed';
-      } else {
-        event.meta.status = 'completed';
-      }
-      const granuleId = get(event, 'meta.granuleId', null);
-      if (granuleId) {
-        await setGranuleStatus(
-          granuleId,
-          event.meta.stack,
-          event.cumulus_meta.system_bucket,
-          event.cumulus_meta.state_machine,
-          event.cumulus_meta.execution_name,
-          event.meta.status
-        );
-      }
-    } else {
-      event.meta.status = 'running';
-    }
-
-    const sns = new AWS.SNS();
-    await sns.publish({
-      TopicArn: topicArn,
-      Message: JSON.stringify(event)
-    }).promise();
-  }
+  await publishReportSnsMessages(event, finished, failed);
```

Comment:
I plan to delete this Lambda at some point, as it is entirely redundant of `sf-sns-report`, however the `KinesisTestTriggerSpec` currently relies on it. So I refactored it for consistency with the updated `sf-sns-report`

Reply:
I was planning to do it as part of 1396. Since 1396 involves removing the start/stop status tasks, which is what the test is relying on, so it will have to be refactored anyway.